### PR TITLE
Endpoint renaming

### PR DESCRIPTION
**Pull request by mattheworris**
Swarm on synchronizing endpoints: built on top of k6 load tests.
Pair reviewed and merged.

### CODE REVIEW

File: apps/api/src/controllers/api.controller.ts [hunk 1]
```diff
@@ -2,8 +2,8 @@ import { ApiService } from '#api/services/api.service';
 import { Controller, Get, HttpCode, HttpStatus, Logger } from '@nestjs/common';
 import { ApiOkResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
 
-@Controller('api')
-@ApiTags('api')
+@Controller()
+@ApiTags('account-service', 'health')
```
Comment (wilwade) [inline apps/api/src/controllers/api.controller.ts:6]: This was a switch we made:
```suggestion
@ApiTags('health')
```

File: apps/api/src/controllers/api.controller.ts
Comment (wilwade) [inline apps/api/src/controllers/api.controller.ts:1]: File can be removed. No need to keep the old /health around

File: apps/api/src/controllers/health.controller.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  @ApiOperation({ summary: 'Check the live status of the service' })
+  @ApiOkResponse({ description: 'Service is live' })
+  livez() {
+    return {
+      status: HttpStatus.OK,
+      message: 'Service is live',
+    };
+  }
+
+  // Live endpoint
```
Comment (wilwade) [inline apps/api/src/controllers/health.controller.ts:33]:
```suggestion
  // Ready endpoint
```

Comment (wilwade) [reply apps/api/src/controllers/health.controller.ts:33]: I missed this sorry

File: k6-test/new-sign-up.js [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+// function initiateApiRequest() {
+//   const res = http.post('https://api.example.com/trigger-callback', {
+//     callbackUrl: CALLBACK_URL,
+//   });
+//   check(res, {
+//     'initiation successful': (r) => r.status === 200,
+//   });
+// }
+
```
Comment (mattheworris) [inline k6-test/new-sign-up.js:57]:
```suggestion
```

File: package.json [hunk 4]
```diff
@@ -28,7 +28,7 @@
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
     "pretest": "cp env.template .env",
     "test": "jest  --coverage --verbose",
-    "test:e2e": "set -a ; . .env ; jest --verbose --silent --runInBand --testRegex \".e2e-spec.ts\" --detectOpenHandles --forceExit"
+    "test:e2e": "set -a ; . .env ; jest --runInBand --testRegex \".e2e-spec.ts\" --detectOpenHandles --forceExit"
```
Comment (mattheworris) [inline package.json:31]:
```suggestion
    "test:e2e": "set -a ; . .env ; jest --verbose --silent --runInBand --testRegex \".e2e-spec.ts\" --detectOpenHandles --forceExit"
```

**Review by claireolmstead — APPROVED** [review]
- pair reviewed

🚢 !

**Review by claireolmstead — APPROVED** [review]

**Pull request merged**